src/QMCDrivers/QMCUpdateBase.cpp

```diff
@@ -264,6 +264,46 @@ QMCUpdateBase::RealType QMCUpdateBase::getNodeCorrection(const ParticleSet::Part
   return setScaledDriftPbyPandNodeCorr(Tau, MassInvP, g, gscaled);
 }
 
+bool QMCUpdateBase::checkLogAndGL(ParticleSet& pset, TrialWaveFunction& twf)
+{
+  bool success         = true;
+  TrialWaveFunction::LogValueType log_value{twf.getLogPsi(), twf.getPhase()};
+  ParticleSet::ParticleGradient_t G_saved = twf.G;
+  ParticleSet::ParticleLaplacian_t L_saved = twf.L;
+
+  pset.update();
+  twf.evaluateLog(pset);
+
+  const RealType threshold = 100 * std::numeric_limits<float>::epsilon();
```

Comment:
Odd indentation here. Missed clangformat?

Reply:
I think I forgot applying clang-format. Will get this done. when I update the other PR.